pkg/storaged/utils.js

```diff
@@ -755,3 +755,7 @@ export function fmt_to_array(fmt, arg) {
     else
         return [fmt];
 }
+
+export function reload_systemd() {
+    return cockpit.spawn(["systemctl", "daemon-reload"], { superuser: "require" });
```

Comment:
The callers don't handle any errors from this, so any failure would result in a page oops. Indeed this Should Not Fail™, but maybe one of these days it may if it detects a broken unit? It may be more robust to add `err: "message"` here and log and ignore an error? If it fails, there's not much that we can do anyway.

Reply:
> any failure would result in a page oops.

Failures appear in the dialog, but only as "systemctl exited with code 1", `err: "message"` is indeed necessary.